pages/gigabit/Project.jsx

```diff
@@ -52,12 +60,14 @@ module.exports = React.createClass({
           </div>
         </div>
 
-        <div className="m-b-3 text-center">
-          <p>Learn more about {data.Project}</p>
-          <div className="middle-button">
-            <a className="secondary-button" href={data.Link} target="_blank">Website</a>
+        { data.Links &&
```

Comment:
You are right. I guess it's more of personal coding style. I usually would declare `links` outside of the `if` block.